server/controllers/devices.coffee

```diff
@@ -61,7 +61,7 @@ deviceExists = (login, cb) ->
         else if body.length is 0
             cb null, false
         else
-            cb null, body[0]
+            cb null, body[0]?.value
```

Comment:
I think it is one if these case where things just happened to work : 

body[0] would be `{id: "xxx", key:"xxx", value:{actual doc}}`, we return this in callback as device.
It is truthy, so the `if device` in `checkLogin` will pass, then we do `device.id` in removeDevice, which pass too.

So this is a good change, but i dont think it was this one which fixed the error in travis :)